docs/user/openstack/README.md

```diff
@@ -49,6 +49,94 @@ hang waiting for the console to come up.
 `DEBUG Still waiting for the console route: the server is currently unable to
 handle the request (get routes.route.openshift.io)`
 
+## Using an External Load Balancer
+
+This documents how to shift from the api VM load balancer (which is not
+HA) to an external load balancer.
+
+The load balancer must serve ports 6443, 443, and 80 to any users of
+the system.  Port 49500 is for serving ignition startup configurations
+to the OpenShift nodes and should not be reachable by the outside world.
+
+The first step is to add floating IPs to all the master nodes. Usually
+the public network here is named 'public':
+
+* `openstack floating ip create --port master-port-0 <public network>`
```

Comment:
Is there a way to tag these floating IPs? I wonder if by assigning them manually without tags we'll be breaking the destroy steps. If possible, it'd be great to add the tags to these commands too with a short comment on why it's important to have them.

Reply:
Destroy handles this without tags.  It does it by  using the association to a tagged port//

Reply:
Awesome! Thanks for confirming.